src/ControlzEx/ControlzEx.csproj

```diff
@@ -16,9 +30,12 @@
     </PropertyGroup>
     <!-- Items include -->
     <ItemGroup>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All"/>
         <Reference Include="System.Windows.Interactivity">
             <HintPath>..\ExtLibs\NET45\System.Windows.Interactivity.dll</HintPath>
         </Reference>
     </ItemGroup>
-    <Import Project="..\..\.paket\Paket.Restore.targets"/>
+    <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
```

Comment:
Shouldn't this block be in Direcoty.build.props?
The full framework references are placed there also. ;-)

Reply:
Yeah, moving it